partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
+1 for consistency - I hadn't seen an example like that before, simply displaying an `input-text` group in a panel results in my second example above - do you have another child element somewhere? Could you paste the html from the image above please?